### PR TITLE
Partial parcel ut back-end

### PR DESCRIPTION
**Pull request by iaincrawf**

### CODE REVIEW

File: src/Facade/Services/IFacadeWithSearchReturnTen.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿namespace Linn.Stores.Facade.Services
+{
+    using System.Collections.Generic;
+
+    using Linn.Common.Facade;
+
+    public interface IFacadeWithSearchReturnTen<T, in TKey, in TResource, in TUpdateResource> : IFacadeService<T, TKey, TResource, TUpdateResource>
```
Comment (lewisrenfrew) [inline src/Facade/Services/IFacadeWithSearchReturnTen.cs]: Does this need to be so specific? How about a search method with an option limit parameter that you could set to ten, or anything you like. I'd also include the word 'Service' in the interface name somewhere for conventions sake.

File: src/Facade/Services/SuppliersService.cs [hunk 2]
```diff
@@ -1,24 +1,77 @@
-﻿namespace Linn.Stores.Facade.Services
+﻿//namespace Linn.Stores.Facade.Services
```
Comment (lewisrenfrew) [inline src/Facade/Services/SuppliersService.cs]: not sure what's going on here but I wouldn't let this commented out code into master

Comment (iaincrawf) [reply src/Facade/Services/SuppliersService.cs]: Oops yeah I'll take that out, wasn't going to pr it like that last night then put it up today without checking

File: src/Facade/Services/IFacadeWithSearchReturnTen.cs (hunk 1 above)
Comment (iaincrawf) [reply src/Facade/Services/IFacadeWithSearchReturnTen.cs]: Yeah I can change it to have an optional limit and chuck service in the name, I basically just copied it in from production to use it in a similar way

**Pull request merged**